mainchain-follower/main-chain-follower-api/src/native_token.rs

```diff
@@ -0,0 +1,19 @@
+use crate::Result;
+use async_trait::async_trait;
+use sidechain_domain::*;
+
+pub struct NativeTokenManagementReleaseEvent {
+	pub token_amount: u64,
+}
+
+#[async_trait]
+pub trait NativeTokenManagementDataSource {
+	/// Retrieves all native token releases into the illiquid supply in the range (after_block, to_block]
```

Comment:
There's no metadata attached to the illiquid supply utxos, so we can just query for the sum of their tokens